packages/core/src/components/essentials/legend.ts

```diff
@@ -132,7 +123,36 @@ export class Legend extends Component {
 			addedLegendItemsText.html((d) => d.name);
 		}
 
-		this.breakItemsIntoLines(addedLegendItems);
+		const additionalItems = Tools.getProperty(
+			this.configs,
+			'additionalItems'
+		);
+
+		// Add additional labels
+		if (additionalItems && dataGroups.length) {
+			const additionalLabelItems = svg
+				.selectAll('g.additional-label')
+				.data(additionalItems);
+
+			const addedAdditionalLabelItems = additionalLabelItems
+				.enter()
+				.append('g')
+				.classed('additional-label', true);
+
+			addedAdditionalLabelItems
+				.append('g')
+				.classed('icon', true)
+				.html((d) => d.icon);
```

Comment:
the user would have to add their own aria-label and role to the icon html but maybe add it to the demo code as best practice